packages/forms/frontend/sirius-components-widget-reference/src/components/MultiValuedReferenceAutocomplete.type.ts

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import React from 'react';
+import { GQLReferenceValue, GQLReferenceWidget } from '../ReferenceWidgetFragment.types';
+
+export interface MultiValuedReferenceAutocompleteProps {
+  editingContextId: string;
+  formId: string;
+  widget: GQLReferenceWidget;
+  readOnly: boolean;
+  onDragEnter: (event: React.DragEvent<HTMLDivElement>) => void;
+  onDragOver: (event: React.DragEvent<HTMLDivElement>) => void;
+  onDrop: (event: React.DragEvent<HTMLDivElement>) => void;
+  onMoreClick: (event: React.MouseEvent<Element, MouseEvent>) => void;
+  editReference;
```

Comment:
Why `editReference` does not have a type?

Reply:
Done